ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__sno-spoke-4.18-nightly.yaml

```diff
@@ -28,15 +28,14 @@ resources:
 tests:
 - as: telcov10n-sno-spoke-f360
   cluster: build05
-  cron: '* 4,8,12,16,20 * * *'
+  cron: '* 8,15,22 * * *'
```

Comment:
@natifridman,

By now, I have reduced the cadence to 3 times per day. Once the job is stable enough I'll decrease it even more since I need to test it more frequently yet.